protocol.privnet.json

```diff
@@ -27,7 +27,7 @@
 
   "ApplicationConfiguration": {
     "DataDirectoryPath": "./Chains/privnet",
-    "NotificationDataPath": "Chains/Priv_Notif",
+    "NotificationDataPath": "Chains/privnet_notif",
```

Comment:
I changed this so we can use `rm -rf Chains/privnet*`

Reply:
Perhaps this should be mentioned explicitly in the CHANGELOG since existing `Priv_Notif` chains will no longer be used?

Reply:
absolutely, added right now